src/applications/vaos/reducers/newAppointment.js

```diff
@@ -264,6 +267,12 @@ export default function formReducer(state = initialState, action) {
         showTypeOfCareUnavailableModal: false,
       };
     }
+    case CLICKED_UPDATE_ADDRESS_BUTTON: {
+      return {
+        ...state,
+        clickedUpdateAddressButton: true,
```

Comment:
I think "CLICKED_UPDATE_ADDRESS_BUTTON" is a good name for your action, but I would probably name the state variable here something different. Something like `hideUpdateAddressAlert` instead of `clickedUpdateAddressButton`.

Generally, action names should be past tense, and describe what a user did. So what you chose there is good. But the property names in the Redux state should tell us something about the current state of the application. Just from looking this action and the data that it's changing, we don't know much about what effect this change has on on our application. But if the name is something like `hideUpdateAddressAlert`, then this code tells you that clicking on the update address alert button hides the update address alert, without having to go look up where `clickedUpdateAddressButton` is used.

Reply:
Thanks for explaining the redux naming convention.  Done.